js/src/views/Dapps/dapps.css

```diff
@@ -25,6 +25,6 @@
 
 .item {
   display: inline-block;
-  margin: 0 0 0.25em;
-  width: 100%;
+  margin: 0.25%;
```

Comment:
We have the margins at 0.25em on the accounts list, would like to keep consistency throughout